src/swig/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 if (POLICY CMP0078)
   cmake_policy(SET CMP0078 NEW)
 endif ()
+# clean up old generated source files before running SWIG, useful for Java
+set(UseSWIG_MODULE_VERSION 2)
```

Comment:
this is breaking the Python install rules because the locations are changed

Reply:
Hmmm, I noticed that the Python tests were failing, but I was under the (possibly false) impression that they had already started failing recently before my changes. I am going to double-check where the failure comes from and look into fixing it as soon as possible either way, then updating the pull request.

Reply:
But we may be talking about different things. What exactly is the issue with the Python bindings that you are seeing? Failing tests? `make install` failing? `make install` installing to a wrong location?

Reply:
I guess it is this hardcoded path in `${CMAKE_CURRENT_BINARY_DIR}` which is the issue for you:
```cmake
install (FILES ${CMAKE_CURRENT_BINARY_DIR}/nlopt.py DESTINATION ${INSTALL_PYTHON_DIR})
```
I will fix that ASAP and update the pull request (with this and the other fixes).

Reply:
This should be fixed in the new revision (commit 32a4a64e7b087fb0cec719ccd794ed5a8fe2f585). Both the Python tests and `make install` work now.